### ISSUE TITLE
RSS feed does not work

### ISSUE THREAD

**Issue by cmirnow** [eworkers/K2ForJ4#51]
RSS feed does not work
It looks like if Joomla menu item is formed by K2 tag, and rss feed is enabled in menu item settings - it doesn't work. 

```
<error>
<code>0</code>
<message>Call to a member function get() on string</message>
</error>

```
Click on the rss icon [here](https://masterpro.ws/ruby).

**Comment by eworkers** [comment]
is this a k2forj4 issue specific? or the issue exist with k2 official version on j310 when cache is enabled? knowing this can be extremely helpful to mitigate.

**Comment by cmirnow** [comment]
This problem seems to be specific to K2ForJ4. 
https://ce26335.tw1.ru/ruby?format=feed
(K2, Joomla 3)

**Comment by eworkers** [comment]
did you try to refresh?

**Comment by cmirnow** [comment]
Yeah, that's a good idea... I just upgraded from K2 to K2ForJ4. After deleting the cache the feed works.
Conclusion: the problem appears after upgrading from Joomla 3 to Joomla 4.

**Comment by eworkers** [comment]
thank you again, but the question was if problem exists with official k2 version on j310+, we seem to be able to replicate using k2 official version on joomla 3.10+ < 4.0, can you confirm the replication using official k2 on j310+ < 4.0 ?

**Comment by cmirnow** [comment]
I can only assert the following:

1) K2 and K2ForJ4 do not have this problem on the Joomla 3 platform.
2) After upgrading to Joomla 4 this problem appears.

I don't have any other extrapolations.

**Comment by eworkers** [comment]
closed in 630a3a7
